### PR TITLE
Handle new suitesparse version

### PR DESCRIPTION
**Pull request by AlttiJokinen**
https://swift-nav.atlassian.net/browse/OC-729

Handle directory structure of new Suitesparse version

### CODE REVIEW

File: Findsuitesparse.cmake [hunk 1]
```diff
@@ -18,13 +18,13 @@ endif()
 find_path(suitesparse_cholmod_INCLUDE_DIR
   NAMES cholmod.h
   HINTS ${SUITESPARSE_DIRS}
-  PATH_SUFFIXES include
```
Comment (jbangelo) [inline Findsuitesparse.cmake:21]: I think we need to keep the `include` suffix for the Linux installs and older versions on Mac

Comment (AlttiJokinen) [reply Findsuitesparse.cmake:21]: It seems that I made mistake, it should have been include and not suitesparse

**Comment by AlttiJokinen** [comment]
@ljbade  I can see that your approval is needed, because must be someone from dev infra

**Comment by ljbade** [comment]
Has this now been tested on both Linux and Mac?

**Comment by AlttiJokinen** [comment]
@ljbade  Yes, it has been compiled as part of Albatross pipeline https://github.com/swift-nav/albatross/pull/473#issuecomment-1888240490

**Review by ljbade — APPROVED** [review]

**Pull request merged**